docs/stock/data-grouping.md

```diff
@@ -15,3 +15,17 @@ By default, the grouping [approximation](https://api.highcharts.com/highstock/pl
 Grouping is activated when there are many data points in the chart. As well as increasing performance it makes it easier to spot trends in a chart.
 
 Data grouping is a Highcharts Stock feature and is enabled by default. To see dataGrouping options see the [API reference](https://api.highcharts.com/highstock/plotOptions.series.dataGrouping).
+
+### The x-value for a grouped point
+
+ <iframe width="320" height="800" src="https://jsfiddle.net/gh/get/library/pure/highcharts/highcharts/tree/master/samples/stock/plotoptions/series-datagrouping-first-anchor" allow="fullscreen"></iframe>
+
+Grouped points can be positioned inside the group using [anchor](https://api.highcharts.com/highstock/plotOptions.series.dataGrouping.anchor) property which might take values: 
+
+* `start` places the point always at the beginning of the group  (e.g. range 00:00:00 - 23:59:59 -> 00:00:00)
+* `middle` places the point always in the middle of the group (e.g. range 00:00:00 - 23:59:59 -> 12:00:00)
+* `end` places the point always at the end of the group (e.g. range 00:00:00 - 23:59:59 -> 23:59:59)
+
+Additionally [firstAnchor](https://api.highcharts.com/highstock/plotOptions.series.dataGrouping.firstAnchor) and [lastAnchor](https://api.highcharts.com/highstock/plotOptions.series.dataGrouping.lastAnchor) properties can be used to position the first and the last point in the data set (not in the current zoom). Apart from the the values desribed above, they might have values:
```

Comment:
"Apart from ..." => more precise to say "In addition to..."
Typo: "described" => "described"